lib/voyager-message/src/rpc/server.rs

```diff
@@ -60,8 +57,8 @@ pub struct ServerInner {
 // }
 
 // impl IbcStateCacheKey<'static> {
-//     pub fn new_owned(chain_id: ChainId<'static>, path: Path, height: Height) -> Self {
-//         Self {
+//     pub fn new_owned(chain_id: ChainId<'static>, path: Path, height: Height)
+// -> Self {         Self {
 //             chain_id,
 //             path: Cow::Owned(path),
 //             height,
```

Comment:
do we keep all this comments?

Reply:
i was keeping it for now because i'll likely use this for more manageable cache keys soon (the only changes here is comment wrapping)